lib/prm/employees/employee_search.ex

```diff
@@ -7,6 +7,8 @@ defmodule PRM.Employees.EmployeeSearch do
     field :party_id, Ecto.UUID
     field :division_id, Ecto.UUID
     field :legal_entity_id, Ecto.UUID
+    field :tax_id, :string
+    field :edrpou, :string
```

Comment:
As i can see, it currently doesn't work, since `settlement_id` is in embed schema, you can't query it like (according to specs): 

```
get conn, legal_entity_path(conn, :index, ["settlement_id": 2])
```

settlement_id won't appear in the changeset and won't affect the search results. I've added appropriate test.